src/applications/lgy/coe/containers/EligibilityApp.jsx

```diff
@@ -39,33 +40,39 @@ const EligibilityApp = props => {
   let content;
 
   if (generateAutoCoeStatus === CALLSTATUS.idle || profileIsUpdating) {
-    content = <LoadingIndicator message="Loading application..." />;
+    content = <va-loading-indictator message="Loading application..." />;
```

Comment:
@jerekshoe so glad to see you using the web component loading indicator here (and below), but unfortunately it looks like you have a typo and the component is probably not loading properly for you.

Reply:
Haha nice catch. I have another pull request out right now involving that file, I’ll be sure to update it there